packages/myst-to-tex/src/index.ts

```diff
@@ -149,7 +150,12 @@ const handlers: Record<string, Handler> = {
       return;
     }
     if (node.visibility === 'remove') return;
-    if (node.data?.tags?.includes('no-tex')) return;
+    if (metadataTags.includes('no-tex')) return;
```

Comment:
Can we expand this to `no-pdf` and implement typst (`no-typst`) while we are here?